src/mmrelay/meshtastic/events.py

```
@@ -0,0 +1,930 @@
+                # Using None for passed_config would skip matrix_rooms initialization
+                connect_future = facade.asyncio.ensure_future(
+                    loop.run_in_executor(
+                        None, facade.connect_meshtastic, facade.config, True
+                    )
+                )
+                facade.reconnect_task_future = connect_future
+                connected_client = await connect_future
+                if connected_client is not None:
+                    facade.logger.info("Reconnected successfully.")
+                    break
+            except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)